tar5.4_tool.py

```diff
@@ -0,0 +1,42 @@
+"""
+    fanction that return thee min's lengh of the argument
+"""
+def find_min(*interleave):
+    min =interleave[0]
+    min_len=len(interleave[0])
+    for item in interleave:
+        if min_len>len(item):
+            min=item
+    return min
+
+"""
+    A function that receives several lists and returns one list in the requested order
```

Comment:
Its a better description, but you should give an example of that requested order, or try to explain it.